packages/react-aria-components/docs/Dialog.mdx

```diff
@@ -156,6 +154,15 @@ import {Popover, OverlayArrow} from 'react-aria-components';
 </DialogTrigger>
 ```
 
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+```css
+.dialog-trigger {
```

Comment:
so this appears in a few places in our docs, is it something we want fixed?
https://stackoverflow.com/questions/17643721/firefox-renders-some-characters-smaller-than-the-font-size

Reply:
I fine with it either way, but we probably don't want do too much browser specific fixes for the RAC examples, adds to the CSS complexity/size when we are really just trying to show how to use the RAC classes/data-attributes

Reply:
opted to remove for now, too much CSS and while it looks a little silly, it's not awful